website/src/theme/ChecklistSummary/index.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import {usePluginData} from '@docusaurus/useGlobalData'
+import styles from './styles.module.css'
+import { isChecked } from '../ArticleChecklist';
+import BrowserOnly from '@docusaurus/BrowserOnly';
+
+function matchesTag(tags, t) {
+    return tags.filter(tag => tag.label == t).length > 0;
+}
+
+function ChecklistItem({title, item}) {
+    const checked = isChecked(title+"_"+item.title)
+    console.log("Checked: "+checked)
```

Comment:
I've seen at least one other `console.log`s in this PR, do we want to keep them or can they be removed?

Reply:
removed. Left one useful one commented in category-listing